src/app/pages/Languages.svelte

```diff
@@ -0,0 +1,198 @@
+<script lang="ts">
+  import LanguageInfo from "../Components/LanguageInfo.svelte";
+  import LanguageSources from "../Components/LanguageSources.svelte";
+  import Sidebar from "../Components/Sidebar.svelte";
+  import Button from "../Components/Button.svelte";
+
+  export let selectedButton: string = "information";
+  export let selectedLanguage: string = "Kwakwala";
+
+  // Mock language data object - this would be read from localstorage/db
+  export let languageInformation = {
+    word_count: "74231",
```

Comment:
Two things: please use camelCase for the property name: `wordCount`. Second, we can implement that dynamic word count functionality like this:

```suggestion
    get wordCount(): number {
        return languageInformation.sources.reduce((sum, source) =>  sum + Number(source.size), 0);
    },
```